apps/web/cypress/tests/notification-editor/create-notification.spec.ts

```diff
@@ -9,6 +9,7 @@ describe('Creation functionality', function () {
     cy.waitLoadTemplatePage(() => {
       cy.visit('/workflows/create');
     });
+    cy.waitForNetworkIdle(500);
```

Comment:
Why do we have to add these, I see that we didn't change anything network related in this PR?

Reply:
The test cases were randomly failing on that part